test/tests/rails4_with_engines.rb

```diff
@@ -6,7 +6,7 @@ class Rails4WithEnginesTests < Test::Unit::TestCase
 
   def expected
     @expected ||= {
-      :controller => 1,
+      :controller => 2,
```

Comment:
Please add a real test. Otherwise it's very hard to tell what broke. [Documentation here](https://github.com/presidentbeef/brakeman/wiki/How-to-Report-a-Brakeman-Issue#preparing-tests).